lib/scout/view/desktop_scout.dart

```diff
@@ -90,7 +90,7 @@ class _DesktopScoutState extends State<DesktopScout> {
                       children: [
                         // Sport filter
                         Text(
-                          'APT List',
+                          'Live Markets',
```

Comment:
We already have a markets section. Change the name over there. From my understanding the scout page shows all the apts. The market page shows all the markets.